src/buffer/out/OutputCell.cpp

```diff
@@ -9,6 +9,10 @@
 #include "../../types/inc/convert.hpp"
 #include "../../inc/conattrs.hpp"
 
+// Misdiagnosis in MSVC 17.11: Referencing global constants in the member initializer

```

Comment:
mark these with `BODGY` since that's how we mark things where we had to work around dumb platform problems

Reply:
can we file this on the compiler folx too?

Reply:
Hopefully it's fixed by this: https://developercommunity.visualstudio.com/t/C26493-false-positive-initializing-a-bas/10710542?q=26493